### ISSUE TITLE
Train a Destillation Network on Traffic Signs for the Blackbox

### ISSUE THREAD

**Issue by Logic1ne** [Marvinho/InformatiCupUndCupper#6]
As descriped in https://arxiv.org/pdf/1503.02531.pdf . 
The CNN is trained on the Traffic Sign Dataset normally, but the loss is not calculated with the labels, but the response from the blackbox:

When the images have passed through the Network, upload the Input Images to the blackbox and calculate the loss as the difference between the Network's output and the Blackbox's response (probably MSE or L1, maybe Cross Entropy Loss, but then the class has to be read from the json).

**Comment by Logic1ne** [comment]
Code Example is in AdvGenNet -> see Issue #7

**Comment by Logic1ne** [comment]
Update:
First try to do this was saturday. The Datasaet is much oo big in relation to the time limits set by the blackbox. One epoch took about a day. 🐈
